Slack: Support use of chat.postMessage (#25818)
grafana/grafana: docs/sources/alerting/notifications.md

> Token | If provided, Grafana will upload the generated image via Slack's file.upload API method, not the external image destination.

Reviewer: ```suggestion
Token | If provided, Grafana will upload the generated image via Slack's file.upload API method, not the external image destination. If you use the [chat.postMessage](https://api.slack.com/methods/chat.postMessage) API, this is also required.
```

Reviewer 2: Sounds good.

If you are using the token for a slack bot, then you have to invite the bot to the channel you want to send notifications and add the channel to the recipient field.

> To use `chat.postMessage` API to post message, you have to set `Url` as `https://slack.com/api/chat.postMessage`, and `Recipient` and `Token`.

Reviewer 2: ```suggestion
To use the `chat.postMessage` API to post message, set the following values: 
- `Url` - `https://slack.com/api/chat.postMessage` 
- `Recipient` -
- `Token` - 
```

Reviewer 2: Please add what the user needs to put in the Recipient and Token fields.

Reviewer: @oddlittlebird I think it's better to amend the _Url_, _Recipient_ and _Token_ field descriptions above, since they already state what the fields should contain. Basically, these fields need to be rethought a bit, since mtanda suggests another way of using this functionality.

I made a suggestion for the _Token_ field documentation, but GitHub won't allow the same for the other two fields. I can make those changes later, through another method.
